library/Core.cpp

```diff
@@ -1645,6 +1645,42 @@ bool Core::InitMainThread() {
     // Init global object pointers
     df::global::InitGlobals();
 
+    if (df::global::game->command_line.original.find("--skip-size-check") == std::string::npos) {
```

Comment:
since `global_table` won't be defined if no globals are defined, it won't get there, but there is the very remote chance of a `symbols.xml` that _has_ `global_table` but does not have `game` so...

i'll also change it to skip the test (instead of failing) when `global_table` is not defined, since that is more compatiible with the current linux symbol generation process